tests/function_corpus.py

```diff
@@ -267,7 +267,7 @@ def make_shape(dims, length):
 
     return target_shape.flatmap(generate_source)
 
-def broadcasted_elementwise_strategy(k: int):
+def broadcasted_elementwise_strategy(k: int, min_value=-10.0, max_value=10.0):
```

Comment:
For division we want to account for positives and negatives in both arguments. What if instead there's a `values` strategy as input that defaults to [-10, 10] like we have now. `values` can either be a strategy or a list of strategies. If its a list it has to have length of exactly `k` (assert this). Then you can make a denominator strategy that generates a float between -10 to 10 but then adds epsilon*sign(x) to it to ensure its not zero. epsilon can be like 1e-3.